app-shells/fish/fish-2.5.0.ebuild

```diff
@@ -20,8 +20,12 @@ RDEPEND="
 DEPEND="
 	${RDEPEND}
 	sys-devel/bc
-	sys-devel/gettext
+	nls? ( sys-devel/gettext )
 "
+src_prepare() {
+	eapply "${FILESDIR}/${P}-honor-linguas.patch"
+	default
+}
```

Comment:
You can get rid of the entire src_prepare() function by adding
```
PATCHES=(
        "${FILESDIR}/${P}-honor-linguas.patch"
)
```
beneath the DEPEND block.